modules/cert-manager-explanation-of-certmanager-cr-fields.adoc

```diff
@@ -0,0 +1,178 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-customizing-api-fields.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="cert-manager-explanation-of-certmanager-cr-fields_{context}"]
+= Explanation of fields in the CertManager custom resource
+
+You can use the `CertManager` custom resource (CR) to configure the following core components of the {cert-manager-operator}:
+
+* Cert-manager controller: You can use the `spec.controllerConfig` field to configure the cert‑manager controller pod.
+* Webhook: You can use the `spec.webhookConfig` field to configure the webhook pod, which handles validation and mutation requests.
+* CA injector: You can use the `spec.cainjectorConfig` field to configure the CA injector pod.
+
+[discrete]
+== Common configurable fields in the CertManager CR for the cert-manager components
+
+The following table lists the common fields that you can configure in the `spec.controllerConfig`, `spec.webhookConfig`, and `spec.cainjectorConfig` sections in the `CertManager` CR.
+
+.Common configurable fields in the CertManager CR for the cert-manager components
+[cols=".^2,.^2,.^6a",options="header"]
+|====
+
+|Field|Type|Description
+
+|`overrideArgs`
+|`string`
+|You can override the supported arguments for the cert-manager components.
+
+|`overrideEnv`
+|`dict`
+|You can override the supported environment variables for the cert-manager controller. This field is only supported for the cert-manager controller component.
+
+|`overrideResources`
+|`object`
+|You can configure the CPU and memory limits for the cert-manager components.
+
+|`overrideScheduling`
+|`object`
+|You can configure the pod scheduling constraints for the cert-manager components.
+
+|====
+
+[discrete]
+== Overridable arguments for the cert-manager components
```

Comment:
Can you remove this [discrete] tag? We are removing these as they do not translate in DITA. Please replace with a [id="heading-text_{context}"] tag instead.